packages/icons-svg/svg/twotone/empty-square.svg

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" standalone="no"?>
```

Comment:
Why we need the file `emtpy-circle` and `empty-square`

Reply:
For the sake to create a checked one I added it.